### PR TITLE
removed calls to old api

### PR DESCRIPTION
**Pull request by renaynay**
resolves GEN-396

### CODE REVIEW

File: deploy/finalization.go [hunk 1]
```diff
@@ -198,7 +158,7 @@ func finalizeNode(node db.Node, details *db.DeploymentDetails, absNum int) error
 	}
 
 	_, err = client.DockerExecd(node,
-		fmt.Sprintf("sh -c 'nibbler --node-type %s --api %s --jwt %s --testnet %s --node %s %s 2>&1 >> /nibbler.log'",
-			details.Blockchain, conf.APIEndpoint, details.GetJwt(), node.TestNetID, node.ID, files))
+		fmt.Sprintf("sh -c 'nibbler --node-type %s --jwt %s --testnet %s --node %s %s 2>&1 >> /nibbler.log'",
```
Comment (renaynay) [inline deploy/finalization.go]: Is this the new desired docker exec command?

Comment (atoulme) [reply deploy/finalization.go]: We will remove nibbler as well soon

Comment (renaynay) [reply deploy/finalization.go]: Will that be a separate gen ticket? Or should I do it in this PR?

Comment (atoulme) [reply deploy/finalization.go]: Separate, I think

**Pull request merged**